src/Core/configuration.ts

```diff
@@ -16,6 +16,12 @@ export interface Configuration {
   /** Selected environment */
   environment: Environment;
 
+  /** Adyen merchant account. */
+  merchantAccount: string;
+
+  /** Merchant name to be displayed on payment summary item. */
+  merchantName: string;
```

Comment:
This fields are not part of `AdyenContext` Configuration:

* `merchantAccount` is only used for API calls. It is exist in scope of "Example App" because it makes requests to backend, but **real live** Adyen integration should not have this property on client side.
* `merchantName` is used for ApplePay only.